api/src/services/fields.ts

```diff
@@ -440,6 +445,10 @@ export class FieldsService {
 				['"null"', 'null'].includes(field.schema.default_value.toLowerCase())
 			) {
 				column.defaultTo(null);
+			} else if (typeof field.schema.default_value === 'boolean') {
+				// Convert boolean values to number (required for MySQL)
+				const default_value = field.schema.default_value === true ? 1 : 0;
```

Comment:
Does this work for Postgres? IIRC their true false is "TRUE" instead of 1

Reply:
Yes, it does - see https://www.postgresql.org/docs/9.1/datatype-boolean.html
Also tested with SQLite